src/main/java/uk/gov/hmcts/darts/event/service/DartsGatewayClient.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.darts.event.service;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import uk.gov.hmcts.darts.event.model.DarNotifyEvent;
+
+@FeignClient(name = "darts-gateway-client",
+    url = "${darts.gateway.url}")
+public interface DartsGatewayClient {
+
+    @PostMapping(value = "${darts.gateway.events-dar-notify-path}",
+        consumes = {MediaType.APPLICATION_JSON_VALUE},
+        produces = {MediaType.APPLICATION_JSON_VALUE}
+    )
+    ResponseEntity<Void> notifyEvent(DarNotifyEvent darNotifyEvent);
```

Comment:
Not sure `notifyEvent` is very descriptive. 
Should it be something like `notifyDarPc`?

Reply:
Dunno - I settled on this tbh because of having the `DarNotifyEvent` parameter :)

Reply:
Renamed to be `darNotify` in commit [3d0af41](https://github.com/hmcts/darts-api/commit/3d0af412dbf791460c6a0c22eb36dd668dc0b0d1)